.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest, windows-latest]
-        toolchain: [stable, 1.64.0]
+        toolchain: [stable, 1.76.0]
```

Comment:
What is the motivation for this change? Your commit message suggests that you intended to update the version that is used for the release build, but we are already using the latest stable version for that. This number here is our Minimum Supported Rust Version (MSRV) and it should be kept as low as possible with the current code. Is there anything that we need from 1.76?

Reply:
Yea, my mistake when I update the dep crates `regex-syntax v0.8.2` need MSRV of 1.65.0, Sorry I'll fix it.